### PR TITLE
Adding sound option to message based on GCM docs

### PR DESCRIPTION
**Pull request by zoranlorkovic**
Based on this docs https://developers.google.com/cloud-messaging/http-server-ref 
sound is added to notification payload.

### CODE REVIEW

**Comment by barryvdh** [comment]
Does this change the sound for existing implementations? Or did it always play the default sound?

File: src/GcmChannel.php [hunk 1]
```diff
@@ -100,7 +101,7 @@ protected function handleFailedNotifications($notifiable, Notification $notifica
             }
 
             $this->events->fire(
-                new NotificationFailed($notifiable, $notification, $this, [
+                new NotificationFailed($notifiable, $notification, get_class($this), [
```
Comment (barryvdh) [inline src/GcmChannel.php:104]: Why this change?

**Comment by zoranlorkovic** [comment]
There wasn't option for sound so can't tell how other did implementations, but it shouldn't break it. It will play default sound if no sound is setup.

File: src/GcmChannel.php (hunk 1 above)
Comment (zoranlorkovic) [reply src/GcmChannel.php:104]: This is for this issue reported:

https://github.com/laravel-notification-channels/gcm/issues/15
